context/FilterContext.jsx

```diff
@@ -0,0 +1,34 @@
+import {
+  createContext, useContext, useMemo, useState,
+} from 'react';
+
+const FilterContext = createContext();
+
+export const useFilterContext = () => useContext(FilterContext);
+
+export function FilterProvider({ children }) {
+  const [availability, setAvailability] = useState(true);
+  const [project, setProject] = useState('all');
+  const [roles, setRoles] = useState([
+    'Developer',
+    'Director of Business Development',
+    'Frontend Designer',
+    'Project Manager',
+    'Technical Director',
+    'UI Designer',
+    'User Experience Strategist',
+    'Projects Director',
+  ]);
```

Comment:
All of this will be refactored as we figure out what the controls are going to give us, but I wanted to have something in here to show that the custom hook was working.